data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+	source "Buccaneer Bay"
+	destination "Hippocrates"
+	passengers 19
+	to offer
+		"combat rating" > 500
+	
+	on offer
+		conversation
+			`It is dusk at the <origin> spaceport, and the setting sun casts long shadows across the spaceport deck. You take notice of a slim cloaked figure walking only a few feet behind you who seems to be keeping pace with you.`
+			choice
+				`	(Confront them.)`
+					goto confront
+				`	(Stop walking and see if they stop as well.)`
+			
+			`	You stop be the entrance to a store that claims to have "the best Polecats!" and lean against the building, trying not to draw attention to yourself. The figure keeps walking at the same pace, and as they pass you a soft voice says "Follow me," from under the cloak before the figure bolts off into a dark alleyway next to the building.`
+				goto choice
+			
+			label confront
+			`	You turn around and stare right at the figure, who stops within a foot of you. The shadows cast by the sun make it impossible for you to make out their face under the cloak. "Follow me," a soft voice says from under the cloak before the figure bolts off into a dark alleyway next to the building`
+			
+			label choice
+			choice
+				`	(Follow the figure.)`
+				`	(Return to my ship.)`
+					decline
+			
+			`	You step into the alleyway and find the figure standing under a lamp. The figure removes their cloak, revealing the face of a woman with short dark hair and a distinct jagged scar across her right cheek. She begins whispering to you. "You're not from this world. I can tell. You're too clean and unscarred. Would you be able to help me?"`
+			choice
+				`	"What do you want?"`
+					goto need
+				`	"Why would I help a pirate?"`
+			
+			`	The woman chuckles. "Not everyone who lives on a pirate world is a pirate, you know. Maybe you need to learn what a stereotype is. Regardless..." she trails off.`
+			
+			label need
+			`	"It isn't what I want, it's what I need," she whispers, putting heavy emphasis on the word "need."`
+			`	"You undoubtedly know about the horrendous slave trade that still exists on pirate worlds today. A slave trade that the Republic doesn't care to abolish, leaving innocent people to waste away like rats in a cage, working for their lives. As if the politicians of the Republic aren't vermin themselves... I'm part of a group of people who have decided to take matters into our own hands. We're mostly people who grew up on pirate worlds, but some of us were born elsewhere."`
+			`	"I assasinated a pirate warlord a couple of days ago and I have been sheltering his slaves ever since. What I need from you is to transport the <bunks> people in my care to <destination>. Another member of my group should be there to pay you <payment> and take the people somewhere far away from here." She cocks her head to the right. "Will you fight for justice and transport these people, Captain?" The look in her eyes suggests that she won't take no for an answer.`
```

Comment:
I think we should put a comma after `days ago`.

I think `from you` should be changed to something like, `(for) you to do`.